File: src/cli/XtFormat.cs
using System.Runtime.InteropServices;

namespace Xt
{
    [StructLayout(LayoutKind.Sequential)]
    public struct XtFormat
    {
        public XtMix mix;
        public XtChannels channels;

        public XtFormat(XtMix mix, XtChannels channels)
        {
            this.mix = mix;
            this.channels = channels;
        }
    }
}

File: src/java/com/xtaudio/xt/XtFormat.java
package com.xtaudio.xt;

public final class XtFormat {

    public XtMix mix = new XtMix();
    public XtChannels channels = new XtChannels();

    public XtFormat() {
    }

    public XtFormat(XtMix mix, XtChannels channels) {
        this.mix = mix;
        this.channels = channels;
    }
}